### PR TITLE
allow glob for testing

### LINKED ISSUE #266: question: run a single test?
out of curiosity, how do you run a single test? either I'm missing something or it seems to be all or nothing..

### PR DESCRIPTION
**Pull request by gmarinov**
close #266 - allows running selected test suites by command line argument

examples:

```
npm test -- --glob=api-player-spec.js
```

```
npm run test-watch -- --glob=api-player-spec.js
```

```
npm test -- --glob=waveform*.js
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ docs
 node_modules
 peaks.js
 peaks.js.map
+.idea/
```
Comment (chrisn) [inline .gitignore]: This seems to be specific to your development environment. Please use your [global .gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) instead.

File: karma.conf.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 'use strict';
+/* eslint-disable */
```
Comment (chrisn) [inline karma.conf.js]: We should prefer to fix any lint errors, rather than disabling linting.

File: test/unit/api-player-spec.js [hunk 3]
```diff
@@ -33,7 +34,8 @@ describe('Peaks.player', function() {
 
     it('should return an updated time if it has been modified through the audio element', function(done) {
       p.on('player_seek', function(currentTime) {
-        var diff = Math.abs(p.player.getCurrentTime() - newTime);
+        // var diff = Math.abs(p.player.getCurrentTime() - newTime);
```
Comment (chrisn) [inline test/unit/api-player-spec.js]: This test is intended to check `player.getCurrentTime()`. We should add separate test to also check `currentTime` as reported in the `player_seek` event.

Comment (gmarinov) [reply test/unit/api-player-spec.js]: ugh, sorry, I didn't notice this, it was just me testing..

File: .gitignore (hunk 1 above)
Comment (gmarinov) [reply .gitignore]: TIL, thanks

File: karma.conf.js (hunk 2 above)
Comment (gmarinov) [reply karma.conf.js]: I assumed this, being outside `src/` would be exempt. fixed

Comment (chrisn) [reply karma.conf.js]: Ah, I think the eslint command in package.json isn't picking it up. I'll fix this.

Comment (gmarinov) [reply karma.conf.js]: in f8b9712 I modified it slightly so the linter stopped complaining (at least on mine)

File: test/unit/api-player-spec.js (hunk 3 above)
Comment (gmarinov) [reply test/unit/api-player-spec.js]: (p.s. reverted)

**Pull request merged**